pages/service/fractional-cto.html

```diff
@@ -82,16 +82,22 @@ <h2 class="u-trailer u-text-bold">What Does a Fractional CTO Do?</h2>
         </div>
       </div>
     </div>
-  </section>
-  <div class="container">
-    <div class="simple-block u-bg-gray">
-      <div class="width-75rem">
-        <h2 class="u-text-bold text-left padding-top-2 simple-block-title">Hire a Chief Technology Officer For a
-          Fraction of the Cost</h2>
-        <img class="text-left width-50 margin-bottom-2 fadein" src="/img/service/fractional_cto_4.jpg"
-             alt="Hire a Chief Technology Officer For a Fraction of the Cost">
+    <div class="card-horizontal overflow-hidden">
+      <div class="row">
+        <div class="small-12 medium-6 padding-horizontal-0 img-wrapper flex-container">
+          <div class="width-25vw">
+            <img class="section-image fadein" src="/img/service/fractional_cto_4.jpg"
```

Comment:
The image above can be reduced to 600x600, I believe @alex-novak

Reply:
@albertvolkman done